lib/assets/javascripts/twine.js.coffee

```diff
@@ -315,7 +315,7 @@ setupAttributeBinding('innerHTML', 'unsafe-html')
 
 preventDefaultForEvent = (event) ->
   (event.type == 'submit' || event.currentTarget.nodeName.toLowerCase() == 'a') &&
-  Twine.getAttribute(event.currentTarget, 'allow-default') != '1'
+  Twine.getAttribute(event.currentTarget, 'allow-default') in ['false', false, 0, undefined, null]
```

Comment:
maybe we want to make a `Twine.hasAttribute` and use https://developer.mozilla.org/en-US/docs/Web/API/Element/hasAttribute ?